src/collections/_documentation/error-reporting/quickstart.md

```diff
@@ -36,8 +36,6 @@ After you completed setting up a project in Sentry, you’ll be given a value wh
 
 {% include components/platform_content.html content_dir='getting-started-config' %}
 
-Most SDKs will now automatically collect data if available, some require some extra configuration as automatic error collecting is not
```

Comment:
Removed this since we should not be just giving a generic 'most SDKs will', we should explicitly call out in each SDK whether it collects errors automatically or not.